app/experimenter/kinto/tasks.py

```diff
@@ -45,6 +47,19 @@ def push_experiment_to_kinto(experiment_id):
     try:
         client.push_to_kinto(data)
 
+        experimenter_kinto_user, _ = get_user_model().objects.get_or_create(
+            email=settings.KINTO_DEFAULT_CHANGELOG_USER,
+            username=settings.KINTO_DEFAULT_CHANGELOG_USER,
+        )
+        ExperimentChangeLog.objects.create(
+            experiment=experiment,
+            old_status=experiment.status,
+            new_status=experiment.status,
+            message="Recipe Sent to Kinto",
+            changed_values=data,
```

Comment:
Yeah I think we can just wrap this in an outer object that makes it mirror the structure we use in the other ones so that later when we visualize it it's all consistent, something like

```py
{'recipe': {
  'new_value': data,
  'old_value': None,
  'display_name': 'Recipe',
}
```